### PR TITLE
Sylius 1.13

### PR DESCRIPTION
**Pull request by Prometee**
Here is a PR aiming to fix the build and add support for Sylius 1.13. I also:
- [x] Remove support for Sylius 1.11.
- [x] Add Panther config to run Behat.
- [x] Add `symfony/runtime` on the test application.
- [x] Update PHPStan config.
- [x] Update Github action build.
- [x] Fix the Behat build by updating number format on scenarios.

### CODE REVIEW

**Comment by Prometee** [comment]
@Zales0123 @GSadee it seems the build was broken a long time ago but I'm able to run it without any issue on my Mac.

I think the `waitFor` have to be replaced with something else now.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,36 +21,31 @@ jobs:
...
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.11.10", "~1.12.0"]
-                node: ["16.x"]
-                mysql: ["5.7", "8.0"]
-
-                exclude:
-                    - sylius: "~1.11.10"
-                      symfony: "^6.0"
-
+                php: ["8.1","8.2","8.3"]
```
Comment (GSadee) [inline .github/workflows/build.yml:24]:
```suggestion
                php: ["8.1", "8.2", "8.3"]
```

File: composer.json [hunk 2]
```diff
@@ -19,34 +19,35 @@
         }
     ],
     "require": {
-        "php": "^8.0",
```
Comment (GSadee) [inline composer.json:22]: IMO requirement for PHP version should stay

File: tests/Application/Kernel.php [hunk 3]
```diff
@@ -101,10 +100,17 @@ private function registerBundlesFromFile(string $bundlesFile): iterable
     {
         $contents = require $bundlesFile;
 
-        if (SyliusKernel::MINOR_VERSION > 10) {
+        if (SyliusCoreBundle::MINOR_VERSION > 10) {
             $contents = array_merge(
                 ['Sylius\Calendar\SyliusCalendarBundle' => ['all' => true]],
```
Comment (GSadee) [inline tests/Application/Kernel.php:105]: It could be added without condition

File: tests/Application/config/packages/security.yaml
Comment (GSadee) [inline tests/Application/config/packages/security.yaml:1]: The security file should be different in 1.12 and 1.13 due to changes in API

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Francis! :1st_place_medal:

**Pull request merged**